packages/twenty-front/src/modules/object-record/record-index/options/components/RecordIndexOptionsDropdownContent.tsx

```diff
@@ -158,6 +158,7 @@ export const RecordIndexOptionsDropdownContent = ({
             onClick={() => handleSelectMenu('hiddenFields')}
             LeftIcon={IconEyeOff}
             RightIcon={IconChevronRight}
+            hasRightIcon={true}
```

Comment:
why do you introduce this and not look at RightIcon directly?

Reply:
Thanks! what I ment was `hasRightIcon` seems synonymous of `!isUndefinedOrNull(RightIcon)` no? maybe I'm missing something. But it doesn't seem like a clean component api if you can have rightIcon=XXX and hasRightIcon=false (contradiction)